pkg/render/node.go

```diff
@@ -283,6 +304,38 @@ func (c *nodeComponent) nodeRoleBinding() *rbacv1.ClusterRoleBinding {
 	return crb
 }
 
+// nodeRoleBinding creates a clusterrolebinding giving the node service account the required permissions to operate.
+func (c *nodeComponent) CNIPluginRoleBinding() *rbacv1.ClusterRoleBinding {
```

Comment:
nit, but please call this `cniPluginClusterRoleBinding` - roles and clusterroles, and thus rolebindings and clusterrolebindings are different resources.

Reply:
I understand that some of the other functions in this file aren't correct either... e.g., `nodeRole` should be `nodeClusterRole`, but that's just another pre-existing naming bug :sweat:

Reply:
Also, golang uses the casing of the function to determine if it should be public / private.

I don't think this function needs to be public, so it should be lowercase - `cniPluginClusterRoleBinding`